blocks-common/b-page/__csp/b-page__csp.bemhtml

```diff
@@ -0,0 +1,73 @@
+block b-page, elem csp {
+
+    bem: false
+
+    tag: 'meta'
+
+    attrs: {
+        'http-equiv': 'Content-Security-Policy',
+        'content': Object.keys(this.ctx.policies).map(function(name) {
```

Comment:
This template wil never work in browser. According w3c, inserting meta tag in browser doesn't work.